### PR TITLE
[TACHYON-1166] remove TCPRemoteBlockReader

### PR DESCRIPTION
**Pull request by miasantreble**
https://tachyon.atlassian.net/browse/TACHYON-1166

### CODE REVIEW

File: docs/Configuration-Settings.md [hunk 1]
```diff
@@ -526,9 +526,8 @@ The user configuration specifies values regarding file system access.
 <tr>
   <td>tachyon.user.block.remote.reader.class</td>
   <td>tachyon.client.netty.NettyRemoteBlockReader</td>
-  <td>Selects networking stack to run the client with. Valid options are
-    `tachyon.client.netty.NettyRemoteBlockReader` (read remote data using netty) and
-    [DEPRECATED] `tachyon.client.tcp.TCPRemoteBlockReader`</td>
+  <td>Selects networking stack to run the client with. Valid options include
+    `tachyon.client.netty.NettyRemoteBlockReader` (read remote data using netty).</td>
```
Comment (calvinjia) [inline docs/Configuration-Settings.md]: Could you reword this to like `Currently only ... is valid`?

**Comment by apc999** [comment]
Jenkins, test this please.

**Comment by apc999** [comment]
+1 LGTM

**Comment by calvinjia** [comment]
LGTM
